### ISSUE TITLE
sync stuck at got lstat error on node

### ISSUE THREAD

**Issue by raffis** [gyselroth/balloon-client-desktop#110]
#### Description
**Is**:
```
INFO <3> [sync-remote-delta]: Applying collections which need to be redownloaded
INFO <3> [sync-remote-delta]: getDelta ended
INFO <3> [sync-delta]: getDelta ended
DEBUG <3> [sync-delta]: Applying grouped delta
INFO <3> [sync.main]: Applying error queue ({"stopped":false})
INFO <3> [sync.main]: Procesing directory changes ({"stopped":false})
INFO <3> [sync]: transfer started
INFO <3> [sync.main]: Processing file changes ({"stopped":false})
INFO <3> [sync.main]: Processing removes ({"stopped":false})
INFO <3> [sync.main]: Processing action queue ({"stopped":false})
INFO <3> [unknown]: FS: mkdir /New Folderde
DEBUG <main> [sync.watcher]: local watcher emmited changes ({"changes":[{"action":0,"directory":"/home/raffael/Balloon","file":"New Folderde"}],"paused":true})
INFO <3> [unknown]: FS: mkdir /New Folderde/New Folderdedee
INFO <3> [unknown]: FS: mkdir /purple folders3
INFO <3> [unknown]: FS: mkdir /test
INFO <3> [unknown]: FS: mkdir /New folder (guiu)
INFO <3> [unknown]: FS: mkdir /New Folderdde
INFO <3> [unknown]: FS: mkdir /New Folderf
INFO <3> [unknown]: FS: mkdir /New folder
INFO <3> [unknown]: FS: mkdir /New Folderd
INFO <3> [unknown]: FS: mkdir /New Folderd/new
INFO <3> [unknown]: FS: mkdir /New Folderd/new/New Folderd
INFO <3> [unknown]: FS: mkdir /New Folderd/New Folderde
INFO <3> [unknown]: FS: mkdir /New Folderd/New Folderdede
INFO <3> [unknown]: FS: mkdir /New folder (j9vf)
INFO <3> [unknown]: FS: rename node /New Folder /New Folderde/New Folder
ERROR <main> [sync.watcher.local]: got lstat error on node ({"nodePath":"/New Folder","change":{"action":3,"directory":"/home/raffael/Balloon","oldFile":"New Folder","newFile":"New Folder"},"code":"ENOENT"})
DEBUG <main> [sync.watcher]: local watcher emmited changes ({"changes":[{"action":0,"directory":"/home/raffael/Balloon/New Folderde","file":"New Folderdedee"},{"action":0,"directory":"/home/raffael/Balloon","file":"purple folders3"},{"action":0,"directory":"/home/raffael/Balloon","file":"test"},{"action":0,"directory":"/home/raffael/Balloon","file":"New folder (guiu)"},{"action":0,"directory":"/home/raffael/Balloon","file":"New Folderdde"},{"action":0,"directory":"/home/raffael/Balloon","file":"New Folderf"},{"action":0,"directory":"/home/raffael/Balloon","file":"New folder"},{"action":0,"directory":"/home/raffael/Balloon","file":"New Folderd"},{"action":0,"directory":"/home/raffael/Balloon/New Folderd","file":"new"},{"action":0,"directory":"/home/raffael/Balloon/New Folderd/new","file":"New Folderd"},{"action":0,"directory":"/home/raffael/Balloon/New Folderd","file":"New Folderde"},{"action":0,"directory":"/home/raffael/Balloon/New Folderd","file":"New Folderdede"},{"action":0,"directory":"/home/raffael/Balloon","file":"New folder (j9vf)"},{"action":3,"directory":"/home/raffael/Balloon","oldFile":"New Folder","newFile":"New Folder"}],"paused":true})
INFO <1> [unknown]: API: GET request to /user/quota-usage with params: 
INFO <1> [unknown]: API: GET request to /user/quota-usage with params: 

... sync stuck here, nothing more happens
```

**Should**:
Handle error correctly.

#### balloon desktop version
0.2.1

#### Your client operating system
Linux Mint

**Comment by raffis** [comment]
fixed due a server issue in the api v1 delta https://github.com/gyselroth/balloon/issues/161

**Comment by raffis** [comment]
We may have to avoid an endless loop like this somehow.

**Comment by pixtron** [comment]
Should be fixed with: b77e625313c7090b86423c7e9e4e2c9d195670c5
